pypolygames/__init__.py

```diff
@@ -25,3 +25,4 @@
 mcts = os.path.join(root, "build/src", "mcts")
 if mcts not in sys.path:
     sys.path.append(mcts)
+
```

Comment:
```suggestion
```

Reply:
This is what `black` would do.

Code style reviews are a waste of time and we should just use a formatter like `black`.

Reply:
Nice idea, how do we set it up?

Reply:
Also I fixed the newline thing, I think we're saying the same thing

Reply:
if you `pip install black`, you run `black <filename>` to lint a file

If you want to add it as a GitHub check too then you do it through CircleCI (`go_attack` does this: https://github.com/AlignmentResearch/go_attack/blob/main/.circleci/config.yml#L59-L76)